docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,256 @@
+
+:::note
+Please be aware that there are multiple CRDs named IPPool within the Harvester HCI ecosystem. To avoid confusion, ensure that you are working with the IPPool CRD belonging to the `network.harvesterhci.io` API group. Another similarly named CRD exists under the `loadbalancer.harvesterhci.io` API group. For the operations against the IPPool CRD relevant to load balancers, please refer to [this page](../../networking/ippool.md).
+:::
+
+For example:
+
+```yaml
+apiVersion: network.harvesterhci.io/v1alpha1
+kind: IPPool
+metadata:
+  name: example
+  namespace: default
+spec:
+  ipv4Config:
+    serverIP: 192.168.100.2 # The DHCP server's IP address
+    cidr: 192.168.100.0/24 # The subnet information, must be in the CIDR form
+    pool:
+      start: 192.168.100.101
+      end: 192.168.100.200
+      exclude:
+      - 192.168.100.151
+      - 192.168.100.187
+    router: 192.168.100.1 # The default gateway, if any
+    dns:
+    - 1.1.1.1
+    domainName: example.com
+    domainSearch:
+    - example.com
+    ntp:
+    - pool.ntp.org
+    leaseTime: 300
+  networkName: default/example # The namespaced name of the NAD object
+```
+
+After the creation of the IPPool object, the controller reconciliation process will initialize the IP allocation module and spawn the agent Pod for the network.
+
+```shell
+$ kubectl get ippools.network example
+NAME      NETWORK           AVAILABLE   USED   REGISTERED   CACHEREADY   AGENTREADY
+example   default/example   98          0      True         True         True
+```
+
+#### VirtualMachineNetworkConfig
+
+The VirtualMachineNetworkConfig CRD resembles a **request for IP address issuance**. It is associated with NetworkAttachmentDefinition objects.
+
+A sample VirtualMachineNetworkConfig object looks like the following:
+
+```yaml
+apiVersion: network.harvesterhci.io/v1alpha1
+kind: VirtualMachineNetworkConfig
+metadata:
+  name: test-vm
+  namespace: default
+spec:
+  networkConfig:
+  - macAddress: 22:37:37:82:93:7d
+    networkName: default/example
+  vmName: test-vm
+```
+
+After the object being created, the controller tries to get unused IP addresses from the IP allocation module for each MAC address recorded. The IP-MAC mapping will then be updated in the VirtualMachineNetworkConfig object and the corresponding IPPool objects.
+
+:::note 
+In normal cases, it is unnecessary to manually create VirtualMachineNetworkConfig objects for virtual machines. vm-dhcp-controller will help create them during the VirtualMachine reconciliation process. The auto-created VirtualMachineNetworkConfig objects will be deleted along with the removal of VirtualMachine objects.
+:::
+
+### Usage
+
+1. Create a [VM Network](../../networking/harvester-network.md#create-a-vm-network) on the Harvester dashboard
+
+    ![](/img/v1.3/vm-dhcp-controller/vm-network.png)
+
+1. Create an IPPool object (currently unavailable via the dashboard; please use the command line tool `kubectl` instead).
```

Comment:
Do we want to kindly mention it's currently unavailable to create IPPool objects on the dashboard of the Harvester UI? Since this is the add-on marked with experimental, we'll have more integration with the Harvester itself (including the backend and frontend) in the future. Eventually, users can create IPPool objects without using the command line tool.

Reply:
I don't see the need to mention a feature that is currently unavailable, especially when we can't provide a release date. The documentation's job is to provide clear instructions to users. In this case, we explicitly tell them to use kubectl, and that should be enough. We can always modify the doc when the Harvester UI finally supports the feature.